src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -126,16 +126,12 @@ public function theStoreHasCurrencyWithExchangeRate($currencyCode, $exchangeRate
     }
 
     /**
-     * @Given /^(that channel) allows to shop using the "([^"]+)" currency$/
+     * @Given /^(that channel) allows to shop using the "([^"]+)" currency(?: also|)$/
      * @Given /^(that channel) allows to shop using "([^"]+)" and "([^"]+)" currencies$/
      * @Given /^(that channel) allows to shop using "([^"]+)", "([^"]+)" and "([^"]+)" currencies$/
      */
     public function thatChannelAllowsToShopUsingAndCurrencies(ChannelInterface $channel, ...$currenciesCodes)
     {
-        foreach ($channel->getCurrencies() as $currency) {
-            $channel->removeCurrency($currency);
```

Comment:
Why should we remove remove all of previous currencies? Is there any particular reason for that? Anyway, we should probably change logic of getCurrencies